lib/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :name, :interests, :supplies
+
+  def initialize(data)
+    @name = data[:name]
+    @interests = data[:interests]
+    @supplies = {}
+  end
+
+  def add_supply(supply)
+    # new_hash = {}
+    supply.map do  |key, value|
+      @supplies[key] = value
+      # new_hash[key] = value
+  #   end
+  #   new_hash.map do |key , value|
+  #     @supplies[key] = value += value
```

Comment:
This is close to what you were looking for if you took in 2 params when defining `add_supply`

```
  def add_supply(supply, value)
    if @supplies[supply]
      @supplies[supply] += value
    else
      @supplies[supply] = value
    end
  end
```